pettingzoo/utils/env.py

```diff
@@ -180,57 +185,65 @@ def agent_iter(self, max_iter: int = 2**63) -> AECIterable:
         """
         return AECIterable(self, max_iter)
 
-    def last(self, observe: bool = True) -> Tuple[ObsType, float, bool, Dict[str, Any]]:
+    def last(
+        self, observe: bool = True
+    ) -> Tuple[ObsType, float, bool, bool, Dict[str, Any]]:
         """
-        Returns observation, cumulative reward, done, info   for the current agent (specified by self.agent_selection)
+        Returns observation, cumulative reward, terminated, truncated, info for the current agent (specified by self.agent_selection)
         """
         agent = self.agent_selection
         observation = self.observe(agent) if observe else None
         return (
             observation,
             self._cumulative_rewards[agent],
-            self.dones[agent],
+            self.terminations[agent],
+            self.truncations[agent],
             self.infos[agent],
         )
 
-    def _was_done_step(self, action: None) -> None:
+    def _was_dead_step(self, action: None) -> None:
         """
-        Helper function that performs step() for done agents.
+        Helper function that performs step() for dead agents.
 
         Does the following:
 
-        1. Removes done agent from .agents, .dones, .rewards, ._cumulative_rewards, and .infos
-        2. Loads next agent into .agent_selection: if another agent is done, loads that one, otherwise load next live agent
+        1. Removes dead agent from .agents, .terminations, .truncations, .rewards, ._cumulative_rewards, and .infos
+        2. Loads next agent into .agent_selection: if another agent is dead, loads that one, otherwise load next live agent
         3. Clear the rewards dict
 
         Highly recommended to use at the beginning of step as follows:
 
         def step(self, action):
-            if self.dones[self.agent_selection]:
-                self._was_done_step()
+            if (self.terminations[self.agent_selection] or self.truncations[self.agent_selection]):
+                self._was_dead_step()
                 return
             # main contents of step
         """
         if action is not None:
-            raise ValueError("when an agent is done, the only valid action is None")
+            raise ValueError("when an agent is dead, the only valid action is None")
 
-        # removes done agent
+        # removes terminated agent
```

Comment:
This comment should be updated to say 'terminated and truncated'.

Reply:
Changed to say "dead". Equivalent to saying "terminated or truncated", but has more consistent nomenclature to the docstring. Open to other suggestions though, I haven't dedicated much thought to terminology.